.docker.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -x
+
+env
+storage-ng-travis-cpp && bash <(curl -s https://codecov.io/bash)
```

Comment:
@ancorgs not yet, I am still trying to make it working, but it is not high priority now.